api/types/mount/mount.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package containerd.v1.types;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+// Mount describes mounts for a container.
+//
+// This type is the lingua franca of ContainerD. All services provide mounts
+// to be used with the container at creation time.
+//
+// The Mount type follows the structure of the mount syscall, including a type,
+// source, target and options.
+message Mount {
+       // Type defines the nature of the mount.
+       string type = 1;
+
+       // Source specifies the name of the mount. Depending on mount type, this
+       // may be a volume name or a host path, or even ignored.
+       string source = 2;
+
+       // Target path in container
+       string target = 3;
+
+       // Options specifies zero or more fstab style mount options.
```

Comment:
The format is still the same. We'll figure out how to map them into what windows needs.